DPG/Tasks/qaEventTrackLite.cxx

```diff
@@ -114,7 +114,7 @@ struct qaEventTrackLite {
   Configurable<float> dcaXYmaxSel3{"dcaXYMaxSel3", 999., "Max dca XY sel3"};
 
   // MC selections
-  Configurable<int> pdgCodeSel{"pdgCodeSel", 2, "pdgCode based particle selection, 1 defines pi,K,p,mu,e, 2 all final-state charged particles including light (hyper)nuclei"};
+  Configurable<int> pdgCodeSel{"pdgCodeSel", 2, "pdgCode based particle selection, PDG code required for particles to have OR: 1 defines pi,K,p,mu,e, 2 all final-state charged particles including light (hyper)nuclei"};
```

Comment:
what's OR?

Reply:
it's not clear, you're right. The way it is implemented one can either select a PDG code or select 1 for pikapr or 2 for pikapr.... I will fix this